README.md

```diff
@@ -75,13 +80,13 @@ production:
 - If you are facing difficulties installing RVM, most probably it is because of an older version of rvm shipped with Ubuntu's desktop edition and updating the same resolves the problem.
 
   Removing RVM
-  ```  
```

Comment:
@ankitkataria Which IDE are you using? Why does add the "blank" at the end of multiple lines?

Reply:
Those blank spaces were already present in the README. I use sublime-text 3 along with a editorconfig that removed the blank spaces in the end.